resources/lang/fr/disposable.php

```diff
@@ -3,78 +3,75 @@
     // Common
     'actions'     => 'Actions',
     'airport'     => 'Aéroport',
-    'all'         => 'Voir Tous',
+    'all'         => 'Afficher tout',
     'altn'        => 'Altn',
     'alternate'   => 'Alternatif',
     'close'       => 'Fermer',
-    'darkmode'    => 'Mode Sombre',
+    'darkmode'    => 'Mode Nuit',
     'dest'        => 'Dest',
     'destination' => 'Destination',
-    'fuel_ob'     => 'Carburant à Bord',
-    'latest'      => 'Derniers',
+    'fuel_ob'     => 'Fuel embarqué',
+    'latest'      => 'Récents',
     'map'         => 'Carte',
-    'notes'       => 'Remarques',
     'orig'        => 'Orig',
     'origin'      => 'Origine',
     'registered'  => 'Enregistré',
-    'show_hide'   => 'Voir / Cacher',
+    'show_hide'   => 'Carte / Liste',
     'total'       => 'Total',
     'weather'     => 'Météo',
-    'pagination'  => 'afficher :first premier au :last du total :total des enregistrements',
+    'pagination'  => 'Rapports :first à :last sur :total soumis',
     // Flights
-    'bid_add'     => 'Ajouter une Enchère',
-    'bid_rem'     => 'Retirer une Enchère',
+    'bid_add'     => 'Ajouter aux projets',
+    'bid_rem'     => 'Retirer des projets',
     'load_acars'  => 'Charger dans vmsAcars',
-    'new_pirep'   => 'Nouveau PIREP Manuel',
+    'new_pirep'   => 'Générer PIREP Manuel',
     'sb_generate' => 'Générer OFP SimBrief',
     'sb_view'     => 'Voir OFP SimBrief',
     // Airport Details
     '100ll_cost'     => 'Coût 100LL',
-    'avg_taxi_times' => 'Moyenne du Temps au Taxi',
-    'gh_cost'        => 'Coût de la Manutention au Sol',
+    'avg_taxi_times' => 'Moyenne de durée Taxi',
+    'gh_cost'        => 'Coût du service au Sol',
     'icao_iata_code' => 'ICAO / IATA Code',
     'jeta1_cost'     => 'Coût JET-A1',
     'mogas_cost'     => 'Coût Mogas',
     // Roster
     'awards'      => 'Récompenses',
-    'only_active' => 'Seuls les membres actifs sont répertoriés. La limite inactif est de :days jours',
+    'only_active' => 'Seuls les membres actifs sont répertoriés. Vous serez radié au bout de :days jours d\'absence.',
     'rank'        => 'Grade',
     // Pireps
-    'fused'  => 'Carburant Utilisé',
-    'lpirep' => 'Dernier détails PIREP',
-    'lrate'  => 'Landing Rate',
+    'fused'  => 'Consommé',
+    'lpirep' => 'Dernier PIREP soumis:',
+    'lrate'  => 'Tx de descente',
```

Comment:
"Tx de touché" maybe ? "Tx de descente" would be like a descent rate and not a landing rate